pinot-spi/src/main/java/org/apache/pinot/spi/config/table/IndexingConfig.java

```diff
@@ -283,6 +284,14 @@ public void setNullHandlingEnabled(boolean nullHandlingEnabled) {
     _nullHandlingEnabled = nullHandlingEnabled;
   }
 
+  public boolean useDefaultValueOnError() {
```

Comment:
from my tests, this has to start with `is`, like `isUseDefaultValueOnError`, in order to add it to TableConfig, otherwise it's discarded and marked as unrecognized unrecognizedProperties.

Reply:
done